src/axom/primal/operators/winding_number.hpp

```diff
@@ -289,13 +296,13 @@ double winding_number(const Point<T, 3>& q,
     return 0;
   }
 
-  const double num = Vec3::scalar_triple_product(a, b, c);
-  if(axom::utilities::isNearlyEqual(num, 0.0, EPS))
+  if(squared_distance(q, tri) < edge_tol * edge_tol)
```

Comment:
A good question! When I was debugging and fiddling with the tolerances, I thought that this could be causing an issue.  Using `squared_distance` did fix some of the problems I was having, but it turns out those problems were *also* fixed by something else I did down the line, so I've reverted it to the cheaper operation.